app/models/plage_ouverture.rb

```diff
@@ -21,6 +21,22 @@ class PlageOuverture < ApplicationRecord
 
   has_many :webhook_endpoints, through: :organisation
 
+  scope :for_lieu, ->(lieu) { where(lieu: lieu) }
+  scope :for_motif, ->(motif_name, organisation_id) { joins(:motifs).where(motifs: { name: motif_name, organisation_id: organisation_id }) }
+  scope :with_agent, -> { includes(:agent) }
+  scope :appear_in_range, lambda { |range|
```

Comment:
`in_date_range` ? ou `relevant_for_date_range`?

Reply:
`in_date_range` voir `in` ?